File: src/SimpleApp/Simple.AdminApplication/SysMng/Entities/SysDicValue.cs
namespace Simple.AdminApplication.SysMng.Entities
{
    /// <summary>字典值</summary>
    public class SysDicValue : DefaultTenantEntity, ITreeEntity
    {
        /// <summary>
        /// 编码
        /// </summary>
        public string Code { get; set; }

        /// <summary>名称</summary>
        public string Name { get; set; }

        public string Value { get; set; }

        /// <summary>备注</summary>
        public string Remark { get; set; }

        public int DicTypeId { get; set; }

        /// <summary>
        /// 全名称
        /// </summary>
        public string FullName { get; set; }

        /// <summary>
        /// 是否启用
        /// </summary>
        public int IsEnable { get; set; }

        /// <summary>
        /// 父级GUID
        /// </summary>
        public Guid? ParentId { get; set; }

        /// <summary>
        /// 排序编码
        /// </summary>
        public string OrderCode { get; set; }

        /// <summary>
        /// 排序全编码
        /// </summary>
        public string OrderFullCode { get; set; }
    }
}

File: vue-vben-admin/apps/web-naive/src/api/entity/SysDicValue.ts
export interface SysDicValue {
  code: string  | null | undefined ;    //  编码
  name: string  | null | undefined ;    //  名称
  value: string  | null | undefined ;
  remark: string  | null | undefined ;    //  备注
  dicTypeId: number  | null | undefined ;
  fullName: string  | null | undefined ;    //  全名称
  isEnable: number  | null | undefined ;    //  是否启用
  parentId: string  | null | undefined ;    //  父级GUID
  orderCode: string  | null | undefined ;    //  排序编码
  orderFullCode: string  | null | undefined ;    //  排序全编码
  tenantId: string  | null | undefined ;    //  租户GUID
  id: string  | null | undefined ;    //  主键
  createdId: string  | null | undefined ;    //  创建人GUID
  createdTime: string  | null | undefined ;    //  创建时间
  creator: string  | null | undefined ;    //  创建人
  updatedTime: string  | null | undefined ;    //  更新时间
  updatedId: string  | null | undefined ;    //  更新人GUID
  updator: string  | null | undefined ;    //  更新人
}
